### ISSUE TITLE
Hand Tracking not working

### ISSUE THREAD

**Issue by jetlius** [brownhci/Portalble#1]
Hi,

I tried the demos, both the built example from the website, as well as installing the examples myself with Unity, but the hand tracking is not working in my device (I am using a Samsung S9, which I believe is supported).

Is there anything I may need to activate first? Thank you

**Comment by jeffhuang** [comment]
Are you using the HelloWorld example? What are you seeing?

**Comment by jetlius** [comment]
Yes, I tried the HelloWorld example, the Grab and the Chicken, all of them give the same results.

The planes are being detected, the UI works, but my hand/wrist/arm does not have the points that indicate they are being detected, and I cannot grab and interact with the virtual objects.

**Comment by jetlius** [comment]
> Are you using the HelloWorld example? What are you seeing?

Yes, I tried the HelloWorld example, the Grab and the Chicken, all of them give the same results.

The planes are being detected, the UI works, but my hand/wrist/arm does not have the points that indicate they are being detected, and I cannot grab and interact with the virtual objects.

**Comment by jeffhuang** [comment]
Can you attach a screenshot? Please make sure your hand is fully in view of the camera

**Comment by jetlius** [comment]
> Can you attach a screenshot? Please make sure your hand is fully in view of the camera

![Screenshot_20220517-160138_PortalbleMediaPipe 1](https://user-images.githubusercontent.com/52201980/168844325-6f738b3a-cf01-41df-a2b5-90e26f38fcb8.jpg)
![Screenshot_20220517-160125_PortalbleMediaPipe 1](https://user-images.githubusercontent.com/52201980/168844391-e41111d8-0573-4131-8b42-bfe2786f5ee3.jpg)

**Comment by jeffhuang** [comment]
Thank you, we are able to reproduce the issue on a Samsung S9. We're looking into it. Thanks for your patience, and for reporting the issue.

**Comment by tutu5110** [comment]
Hi Jetlius, Thanks for pointing that out! It has to do with a specific shader we use to share the Unity's camera texture with MediaPipe that is not supported by S9/S9+. S10+ support this texture without an issue. I will update the description to reflect this. Thanks again!

**Comment by shoukailiang** [comment]
I also encountered this problem in Mi8 and Mi11

**Comment by fweidner** [comment]
I have (it seems) the same problem with a Xiaomi 12 Pro but also with a Pixel 4 XL. Downloaded the repo, followed step 1-4, deploy the HelloWorld scene (with the planet). AR is working but hand tracking is not (on both phones). Building and running the app on a Pixel 6a, AR and hand tracking work fine. Log files are attached (logcat output). Can we pinpoint a cause?

For some reason/hunch, I think it is a config/setup issue bc it's supposed to work on a Pixel 4 (according to the documentation) but does not work on a Pixel 4XL. Any hints appreciated and thanks for open sourcing it :)

[Xiaomi12Pro-notworking.txt](https://github.com/brownhci/portalble/files/10294929/Xiaomi12Pro-notworking.txt)
[Pixel6a-working.txt](https://github.com/brownhci/portalble/files/10294930/Pixel6a-working.txt)
[Pixel4XL-notworking.txt](https://github.com/brownhci/portalble/files/10294931/Pixel4XL-notworking.txt)

**Comment by jeffhuang** [comment]
thanks for investigating, @fweidner

@tutu5110 have you seen the errors in the notworking logs before? It seems especially strange that it doesn't work on the Pixel 4 XL
